sdk/communication/azure-communication-common/README.md

```diff
@@ -79,36 +83,49 @@ The following sections provide several code snippets showing different ways to u
 
 ### Creating a credential with a static token
 
+For short-lived clients, refreshing the token upon expiry is not necessary and `CommunicationTokenCredential` may be instantiated with a static token.
+
 ```java
 CommunicationTokenCredential userCredential = new CommunicationTokenCredential("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjM2MDB9.adM-ddBZZlQ1WlN3pdPBOF5G4Wh9iZpxNP_fSvpF4cWs");
 ```
 
 ### Creating a credential that refreshes with a Callable
 
```

Comment:
NIT : Would it make more sense with below statements : 
For on demand : `### Creating a credential that refreshes on demand with a Callable`
For proactive : `## Creating a credential that refreshes proactively with a Callable`

Reply:
good point, thanks